environments/test/test.tfvars

```diff
@@ -8,10 +8,10 @@ backend_agw_private_ip_address = ["10.48.96.111", "10.48.96.114"]
 data_subscription              = "1c4f0704-a29e-403d-b719-b90c34ef14c9"
 privatedns_subscription        = "1baf5470-1c3e-40d3-a6f7-74bfbce4b348"
 oms_env                        = "nonprod"
-
-shutter_storage = "TODO"
-cdn_sku         = "TODO"
-shutter_rg      = "TODO"
+autoShutdown                   = true
+shutter_storage                = "TODO"
+cdn_sku                        = "TODO"
+shutter_rg                     = "TODO"
```

Comment:
these vars look like they should be removed in a separate PR? we no longer use cdn

Reply:
?